README.md

```diff
@@ -219,19 +222,29 @@ If your application needs routing, the SDK provides a multiple approaches to sec
 Additionally to above, Asgardeo offers a wide range of APIs that you can use to integrate and make use of Asgardeo within your React Application. You can refer to a [detailed API documentation here](/API.md).
 
 ## Sample Apps
-
 Sample React Apps offered by Asgardeo will allow you to take Asgardeo for a spin without having to setup your own application. You can see [how to setup sample apps here](/SAMPLE_APPS.md).
 
-## Contribute
+## Develop
+
+### Prerequisites
+-   `Node.js` (version 10 or above).
+-   `yarn` package manager.
 
+### Installing Dependencies
+The repository is a mono repository. The SDK repository is found in the [lib](/lib/) directory. You can install the dependencies by running the following command at the root.
+
+```bash
+yarn build
+```
+
+
```

Comment:
Shall we remove these lines?

Reply:
Fixed with e93c5a331b49dcc9241ac101dc65a04608f362ef